parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/xcm_config.rs

```diff
@@ -108,7 +112,14 @@ match_types! {
 		MultiLocation { parents: 1, interior: Here } |
 		MultiLocation { parents: 1, interior: X1(_) }
 	};
+	// TODO:check-parameter - (https://github.com/paritytech/parity-bridges-common/issues/2084)
+	// remove this and extend `AllowExplicitUnpaidExecutionFrom` with "or SystemParachains" once merged https://github.com/paritytech/polkadot/pull/7005
+	pub type SystemParachains: impl Contains<MultiLocation> = {
+		// Statemine
+		MultiLocation { parents: 1, interior: X1(Parachain(1000)) }
```

Comment:
Should we list encointer in here in we're planning on following PR 7005?

Reply:
well, for our first usecase AssetHubKusama<>AssetHubPolkadot we dont need encointer here, 
so I would wait for that PR 7005